### PR TITLE
docs: Add Xfrep extension and publications section

### PR DESCRIPTION
**Pull request by colluca**

### CODE REVIEW

**Comment by colluca** [comment]
@paulsc96 There is one last thing I don't understand about the FREP instructions, and it's the `stream_ctl` field which is not reported in the original paper. Do you know what its purpose is? I could add it to the documentation and then it should be complete.

File: mkdocs.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ theme:
   icon:
     repo: fontawesome/brands/github
 
-repo_url: https://github.com/pulp-platform/snitch
-repo_name: pulp-platform/snitch
+repo_url: https://github.com/colluca/snitch
```
Comment (paulsc96) [inline mkdocs.yml:11]: Let's revert this to its original state

**Review by paulsc96 — CHANGES_REQUESTED** [review]
I think that if we add a dedicated publication section, we should also update the main README accordingly (see bottom).

File: mkdocs.yml (hunk 1 above)
Comment (colluca) [reply mkdocs.yml:11]: Good catch, thanks!

**Comment by colluca** [comment]
> I think that if we add a dedicated publication section, we should also update the main README accordingly (see bottom).

This is my proposal, what do you think? I think it is important to emphasize that anyone using the core should cite the Snitch paper, whereas the other works are more specific. I would maybe move the SSR paper also below the separator line.

**Review by paulsc96 — APPROVED** [review]

**Pull request merged**